tools/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -331,9 +329,8 @@ private static CLDRFile load(String fileName, String localeName, InputStream fis
      */
     public CLDRFile loadFromInputStream(String fileName, String localeName, InputStream fis, DraftStatus minimalDraftStatus) {
         CLDRFile cldrFile = this;
-        InputStreamReader reader = new InputStreamReader(fis, Charset.forName("UTF-8"));
         MyDeclHandler DEFAULT_DECLHANDLER = new MyDeclHandler(cldrFile, minimalDraftStatus);
-        XMLFileReader.read(fileName, reader, -1, true, DEFAULT_DECLHANDLER);
+        XMLFileReader.read(fileName, fis, -1, true, DEFAULT_DECLHANDLER);
```

Comment:
I thought you were going to add a comment here (if I understand what you said previously) since this is an important change.